collectd-jenkins/README.md

```diff
@@ -0,0 +1,229 @@
+# ![](./img/integrations_jenkins.png) Jenkins
+
+_This directory consolidates all the metadata associated with the jenkins plugin for collectd. The relevant code for the plugin can be found [here](https://github.com/signalfx/collectd-jenkins)_
+
+- [Description](#description)
+- [Requirements and Dependencies](#requirements-and-dependencies)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage](#usage)
+- [Metrics](#metrics)
+- [License](#license)
+
+### DESCRIPTION
+
+This is the SignalFx Jenkins plugin. Follow these instructions to install the Jenkins plugin for collectd.
+
+The [`collectd-jenkins`](https://github.com/signalfx/collectd-jenkins) plugin collects metrics from jenkins instances hitting these endpoints: [../api/json](https://wiki.jenkins.io/display/jenkins/remote+access+api) (job metrics)  and [metrics/<api_key>/..](https://wiki.jenkins.io/display/JENKINS/Metrics+Plugin) (default and optional Codahale/Dropwizard JVM metrics).
```

Comment:
You should use [collectd-jenkins] in the link description instead of [`collectd-jenkins`]. This is so that these docs can be reused in the Signalfx main docs site with minimal preprocessing. All existing docs are being updated.